restcomm/configuration/config-scripts/as7-config-scripts/restcomm/autoconfig.d/config-restcomm.sh

```diff
@@ -33,25 +33,25 @@ configRestcomm() {
 	static_address="$1"
 
 	sed -e  "s|<\!--.*<external-ip>.*<\/external-ip>.*-->|<external-ip>$static_address<\/external-ip>|" \
-		-e "s|<external-ip>.*<\/external-ip>|<external-ip>$static_address<\/external-ip>|" \
+		-e "s/<external-ip>.*<\/external-ip>/<external-ip>$static_address<\/external-ip>/g;s/<external-ip\/>/<external-ip>$static_address<\/external-ip>/g" \
```

Comment:
dont see the point of replacing sed separator from "|" to "/" ,since XML closing tag was already escaped...

Reply:
is /g the global substituion flag here? as in https://stackoverflow.com/questions/28231360/command-sed-using-g
what is the purpose of using "/g" here?
are we expecting multiple xml tags with same name in the same line?
If its only to add a new substitution match covering the empty xml tag (<tag/>)?  

would this work as well:
`
sed -e  "s|<\!--.*<external-ip>.*<\/external-ip>.*-->|<external-ip>$static_address<\/external-ip>|" \
		-e "s/<external-ip>.*<\/external-ip>/<external-ip>$static_address<\/external-ip>/" \
                -e "s/<external-ip\/>/<external-ip>$static_address<\/external-ip>/" \
$FILE > $FILE.bak;
`